projects/cdk/package.json

```diff
@@ -20,7 +20,7 @@
         "@ng-web-apis/resize-observer": "2.0.0",
         "@tinkoff/ng-event-plugins": "3.1.0",
         "@tinkoff/ng-polymorpheus": "4.0.10",
-        "tslib": "^2.0.0"
+        "tslib": "2.0.0"
```

Comment:
Is renovate bot also in charge of these dependencies?

Reply:
Yes